packages/ngtools/webpack/src/index.ts

```diff
@@ -5,11 +5,5 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-
-export * from './angular_compiler_plugin';
-export * from './interfaces';
-export { ngcLoader as default } from './loader';
-
-export const NgToolsLoader = __filename;
-
+export { AngularPluginOptions, AngularWebpackPlugin, default } from './ivy';
 export * as ivy from './ivy';
```

Comment:
Shouldn't this export be dropped?

Reply:
Marked the namespace export as deprecated.  Also, remove its usage from the builders (in a separate commit).